### PR TITLE
Add soc 129

### PR DESCRIPTION
**Pull request by pinheadmz**
Raw scrape with some tidying up by me. For some reason the LDK pull requests didn't scrape, will have to add those manually.

### CODE REVIEW

File: _posts/2023-10-26-socratic-seminar-129.md [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+    - <!--- TODO replace: [October 11th](https://github.com/lightningnetwork/lightning-rfc/issues/920) --->
+- Bitcoin Contracting Primitives Working Group
+	- <!--- TODO replace: [April 18th](https://github.com/ariard/bitcoin-contracting-primitives-wg/blob/main/meetings/meetings-18-04.md) --->
+
+### Optech
+- [Bitcoin Optech Newsletter #274](https://bitcoinops.org/en/newsletters/2023/10/25/)
+- [Bitcoin Optech Newsletter #273](https://bitcoinops.org/en/newsletters/2023/10/18/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/19/)
+- [Bitcoin Optech Newsletter #272](https://bitcoinops.org/en/newsletters/2023/10/11/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/12/)
+- [Bitcoin Optech Newsletter #271](https://bitcoinops.org/en/newsletters/2023/10/04/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/05/)
+- [Bitcoin Optech Newsletter #270](https://bitcoinops.org/en/newsletters/2023/09/27/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/09/28/)
```
Comment (murchandamus) [inline _posts/2023-10-26-socratic-seminar-129.md:77]: It’s a bit odd that we order e.g. the developer meetings oldest first, but the Newsletters youngest first.

```suggestion
- [Bitcoin Optech Newsletter #270](https://bitcoinops.org/en/newsletters/2023/09/27/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/09/28/)
- [Bitcoin Optech Newsletter #271](https://bitcoinops.org/en/newsletters/2023/10/04/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/05/)
- [Bitcoin Optech Newsletter #272](https://bitcoinops.org/en/newsletters/2023/10/11/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/12/)
- [Bitcoin Optech Newsletter #273](https://bitcoinops.org/en/newsletters/2023/10/18/) [Recap Podcast](https://bitcoinops.org/en/podcast/2023/10/19/)
- [Bitcoin Optech Newsletter #274](https://bitcoinops.org/en/newsletters/2023/10/25/)
```

File: _posts/2023-10-26-socratic-seminar-129.md [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+### [BLIPs](https://github.com/lightning/blips)
+
+
+### [BOLTs](https://github.com/lightningnetwork/lightning-rfc)
+- [BOLT4: Specify max HTLC nLocktime for expiry_too_far](https://github.com/lightning/bolts/pull/1086)
+- [BOLT 1: define what `offered` and `negotiated` mean.](https://github.com/lightning/bolts/pull/1109)
+
+
+## New Releases
+-
```
Comment (murchandamus) [inline _posts/2023-10-26-socratic-seminar-129.md:256]:
```suggestion
- [BDK 0.29.0](https://github.com/bitcoindevkit/bdk/releases/tag/v0.29.0)
- [Bitcoin Core 25.1](https://github.com/bitcoin/bitcoin/releases/tag/v25.1)
- [LDK 0.0.117](https://github.com/lightningdevkit/rust-lightning/releases/tag/v0.0.117)
- [LND v0.17.0-beta](https://github.com/lightningnetwork/lnd/releases/tag/v0.17.0-beta)
```

**Review by murchandamus — COMMENTED** [review]
Thanks for scraping!

File: _posts/2023-10-26-socratic-seminar-129.md (hunk 1 above)
Comment (pinheadmz) [reply _posts/2023-10-26-socratic-seminar-129.md:77]: yeah and PRs are listed top down from most recent merge to oldest. I can flip some of the lists in scraper so there's consistency

File: _posts/2023-10-26-socratic-seminar-129.md [hunk 3]
```diff
@@ -0,0 +1,265 @@
...
+layout: post
+type: socratic
+title: "Socratic Seminar 129"
+meetup: "https://www.meetup.com/bitdevsnyc/events/296435823/"
+---
+
+## Announcements
+Please join us for our next Socratic Seminar. A special thank you to our sponsors [CardCoins](https://cardcoins.co), [Chaincode Labs](https://chaincode.com) and [Wolf NYC](https://wolfnyc.com) for food, refreshments and event space.
+
+If you can't make it to the main event please join us at PUBKEY around 9:30PM. **Learn about this awesome new establishment [here](https://ny.eater.com/2022/12/13/23494423/pubkey-opening-manhattan-bitcoin-bar).**
```
[the remark this answers is not in the document]
Comment (btsea) [reply _posts/2023-10-26-socratic-seminar-129.md:11]: Ya, good idea. I can remove that line.

[the remark this answers is not in the document]
Comment (pinheadmz) [reply _posts/2023-10-26-socratic-seminar-129.md:11]: Ill remove it from the scraper! 

https://github.com/pinheadmz/bitdevs-scraper/blob/940f247659770f3ab275211f1e0d8c1a0765eb00/_template.md?plain=1#L11

Replace with anything?

**Pull request merged**